.gitignore

```diff
@@ -1,16 +1,2 @@
 images/
-node_modules/
-*.log
-yarn.lock
-.eslintcache
-.DS_Store
-.idea/
-static/build
-out/
-
-.env*
```

Comment:
you should always have .env file ignored. Others are also good to have on ignore list.